### DISCUSSION TITLE
[Showcase] Hey Duggee project - Rolly

### DISCUSSION THREAD

**Discussion by pgilfernandez** [friction2d/friction#305] [Show and tell]
This was my very first "serious" project with Friction with the target of being able to export it as an animated SVG. Conclusion: AWESOME!! file size is very small and all animated features got into the final file.

I have to say I've done it with the "experimental" macOS branch and the work was perfect, no glitches, no strange behaviors... Friction worked like a charm, so happy with it.

![loops_v01](https://github.com/user-attachments/assets/9161340c-e966-4d90-b13e-7ddc42815db7)

**Comment by rodlie** [comment]
Nice :smile: 

Always great to see other artists use Friction. My goal as a developer is always to empower artists to create, the more the better.

**Comment by tomgaa** [comment]
Man! This is awesome! :D
Good job!
